spark/src/main/resources/python/zeppelin_pyspark.py

```diff
@@ -120,6 +120,11 @@ def _setup_matplotlib(self):
     # If we don't have matplotlib installed don't bother continuing
     try:
       import matplotlib
+
+      #matplotlib.rcParams['savefig.format'] is only avail in version 1.2.0 and above
+      from distutils.version import LooseVersion
+      if LooseVersion("1.2.0") >= LooseVersion(matplotlib.__version__):
+        return
```

Comment:
Right, agreed in most of the cases if user in interested in using matplotlib library he would have at-least 1.5.x installed if not 2.0.x.

But the idea of this PR is not to support older version, but to make sure pyspark continues to work even is a non-supported version of matplotlib is installed and it should behave like there was no matplotlib installed.